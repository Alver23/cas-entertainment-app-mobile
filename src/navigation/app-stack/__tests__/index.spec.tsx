// Dependencies
import React from 'react';
import { shallow, ShallowRendererProps } from 'enzyme';

// Navigation
import { AppStack } from '../index';

describe('AppStack', () => {
  let component: ShallowRendererProps;

  beforeEach(() => {
    component = shallow(<AppStack />);
  });

  it('should save a snapshot of the component', () => {
    expect(component).toMatchSnapshot();
  });
});
