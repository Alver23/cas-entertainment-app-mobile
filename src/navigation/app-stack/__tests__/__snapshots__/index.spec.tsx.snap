// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppStack should save a snapshot of the component 1`] = `
<UserProvider>
  <ForwardRef(NavigationContainer)
    theme={
      Object {
        "colors": Object {
          "background": "rgb(242, 242, 242)",
          "border": "rgb(216, 216, 216)",
          "card": "#0B132A",
          "notification": "rgb(255, 59, 48)",
          "primary": "#ffffff",
          "text": "#ffffff",
        },
        "dark": false,
      }
    }
  >
    <StackNavigator>
      <Screen
        component={[Function]}
        name="Home"
        options={
          Object {
            "headerShown": false,
          }
        }
      />
      <Screen
        component={[Function]}
        name="artists"
        options={
          Object {
            "headerTitle": "",
          }
        }
      />
      <Screen
        component={[Function]}
        name="students"
      />
      <Screen
        component={[Function]}
        name="groups"
      />
      <Screen
        component={[Function]}
        name="itineraries"
      />
      <Screen
        component={[Function]}
        name="teachers"
      />
      <Screen
        component={[Function]}
        name="users"
      />
      <Screen
        component={[Function]}
        name="artistDetail"
        options={
          Object {
            "headerTitle": "",
          }
        }
      />
    </StackNavigator>
  </ForwardRef(NavigationContainer)>
</UserProvider>
`;
